course_info/static/course_info/js/controllers/DetailsController.js

```diff
@@ -110,14 +110,24 @@
                 });
 
         };
-
+        dc.getCourseDescription = function(course) {
```

Comment:
Discussed with Chrissy and the ILE and SB courses should also display 'Untitled Course' when no title exists.